gnosis/conditional-markets-interface
# Feature #81: 1st Trade (Approval) Modal

# Description

This PR implements changes to how transactions are processed in the frontend, so that they can be approved individually accompanied by a modal explaining the different transaction types.

# Code
This PR will deprecate the previous `asWrappedTransaction` and replace it with a `stageTransaction` method in the Root-Component. This function will take a list of transactions with `name`, `description`, `precheck`-function, `commit`-function and `cleanup`-function to form individual transactions, some of which can be skipped (using precheck function) and some of which can handle state-resets 

# To Test
- Make your first transaction with an account (whitelisted) and notice the modal opening
  - Should only allow first transaction to be "submit"-able
- Next transaction should not open the modal and instead just run the transaction
